crates/re_renderer/src/view_builder.rs

```diff
@@ -124,6 +124,73 @@ pub enum Projection {
     },
 }
 
+impl Projection {
```

Comment:
the `aspect_ratio` parameter here is super confusing. It's only used for `Projection::Orthographic` since `Projection::Perspective` comes with its own. Orthographic projection should instead just do the more classic thing and require not only `vertical_world_size` but also `horizontal_world_size`